PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/src/main/java/com/microsoft/azure/toolkit/intellij/azure/sdk/buildtool/HardcodedAPIKeysAndTokensCheck.java

```diff
@@ -0,0 +1,116 @@
+package com.microsoft.azure.toolkit.intellij.azure.sdk.buildtool;
+
+import com.intellij.codeInspection.LocalInspectionTool;
+import com.intellij.codeInspection.ProblemsHolder;
+import com.intellij.psi.JavaElementVisitor;
+import com.intellij.psi.PsiElement;
+import com.intellij.psi.PsiElementVisitor;
+import com.intellij.psi.PsiNewExpression;
+import org.jetbrains.annotations.NotNull;
+import org.json.JSONObject;
```

Comment:
Can we take a dependency on `azure-json` here and use that instead?

Reply:
I'm working on a centralised json loader for all rules that uses azure-json:)

Reply:
this is the PR for the json loader :) https://github.com/Njeriiii/azure-tools-for-java/pull/16